ScanCommand/build.gradle

```diff
@@ -0,0 +1,154 @@
+/*
+ * Copyright (c) 2024, WSO2 LLC. (https://www.wso2.com).
+ *
+ * WSO2 LLC. licenses this file to you under the Apache License,
+ * Version 2.0 (the "License"); you may not use this file except
+ * in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+plugins {
+    id 'java'
+    id 'application'
+    id 'java-library'
+    id 'checkstyle'
+    id "com.github.spotbugs" version "${spotbugsPluginVersion}"
+    id "de.undercouch.download" version "${downloadPluginVersion}"
+}
+
+group = 'io.ballerina.scan'
+version = "${scanToolVersion}"
+
+repositories {
+    mavenLocal()
+    mavenCentral()
+
+    maven {
+        url = uri('https://repo.maven.apache.org/maven2/')
+    }
+
+    maven {
+        url = 'https://maven.pkg.github.com/ballerina-platform/*'
+        credentials {
+            username System.getenv("packageUser")
+            password System.getenv("packagePAT")
+        }
+    }
+}
+
+dependencies {
+    implementation group: 'info.picocli', name: 'picocli', version: "${picoCLIVersion}"
+
```

Comment:
What's the rational behind having empty lines between the `implementation` dependencies? Shall we group them together without spaces?

Reply:
+1, Fixed in [1d1ebf0](https://github.com/ballerina-platform/static-code-analysis-tool/pull/2/commits/1d1ebf0fa12d0309218bc14f5c1193aef7682024)